changelog.dd

```diff
@@ -225,6 +227,26 @@ assert([4, 7, 5].enumerate.maxElement!`a.value` == tuple(1, 7));
 -------
 )
 
+$(LI $(LNAME2 quantize, $(XREF math, quantize) was added, for rounding to the nearest
```

Comment:
Ditto. Don't put XREF in LNAME2. We have to find another spot for it. Maybe shorten the summary line and put the link + description in a separate paragraph, like so:

    $(LI $(LNAME2 `math.quantize` was added.)
        $(P $(XREF math, quantize) rounds to the nearest multiple of some number.)
        ...
    )

Of course, keep the `$(RELATIVE_LINK2 ...)` in sync with this. I.e., shorten it as well.